game-core/src/main/java/games/strategy/engine/lobby/client/login/LobbyPropertyFileParser.java

```diff
@@ -25,7 +24,7 @@
   @VisibleForTesting
   static final String YAML_PORT = "port";
   @VisibleForTesting
-  static final String YAML_HTTP_SERVER_URI = "http_server_uri";
+  static final String YAML_HTTPS_PORT = "http_server_uri";
```

Comment:
I think you missed to change the string here

Reply:
thanks fixed